layouts/dashboard-layout.vue

```diff
@@ -0,0 +1,306 @@
+<script lang="ts" setup>
+import {
+  Dialog,
+  DialogPanel,
+  Menu,
+  MenuButton,
+  MenuItem,
+  MenuItems,
+  TransitionChild,
+  TransitionRoot,
+} from '@headlessui/vue';
+import {
+  Bars3Icon,
+  PencilSquareIcon,
+  ChatBubbleBottomCenterIcon,
+  Cog6ToothIcon,
+  EnvelopeIcon,
+  IdentificationIcon,
+  PhotoIcon,
+  TrashIcon,
+  ShoppingBagIcon,
+  XMarkIcon,
+  Cog8ToothIcon
+} from '@heroicons/vue/24/solid';
+import { ChevronDownIcon } from '@heroicons/vue/20/solid';
+import { authStore } from '~/store/auth';
+import { useStateStore } from '~/store/stateStore';
+
+interface NavigationItem {
+  name: string;
+  href: string;
+  active: string[];
+  icon: any;
+}
+
+interface UserNavigationItem {
+  name: string;
+  href: string;
+}
+
+const route = useRoute();
+const { cardId } = storeToRefs(useStateStore());
+const { user } = authStore();
+const sidebarOpen = ref(false);
+
+const navigation: NavigationItem[] = [
+  { name: 'Cards', href: '/dashboard', active: ['/dashboard', '/cards/create', `/cards/${cardId.value}/edit`], icon: IdentificationIcon },
+  { name: 'Email Signature', href: '#', active: [], icon: EnvelopeIcon },
+  { name: 'Virtual Backgrounds', href: '#', active: [], icon: PhotoIcon },
+  { name: 'Accessories', href: '#', active: [], icon: ShoppingBagIcon },
```

Comment:
remove Email Signature, Virtual Backgrounds and Accessories

Reply:
Already Removed